custom_model_runner/datarobot_drum/drum/language_predictors/java_predictor/src/main/java/com/datarobot/drum/BasePredictor.java

```diff
@@ -31,6 +31,15 @@ public BasePredictor setName(String name) {
     */
     public abstract String predict(byte[] inputBytes) throws Exception;
 
+
+    /**
+    * Make predictions on input scoring data.
+    * @param inputBytes Input data as binary -> byte[].
+    * @return can technically return anything but DRUM expects either String or byte[].
+    */
+    public abstract <T> T predict_unstructured(byte[] inputBytes, String mimetype, String charset, Map<String, String> query) throws Exception;
```

Comment:
```suggestion
    public abstract <T> T predictUnstructured(byte[] inputBytes, String mimetype, String charset, Map<String, String> query) throws Exception;
```

to follow Java/Scala naming conventions

Reply:
done.

Reply:
thanks